modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
@friedmud , when you say 'the misfit from the original residual (before the preset has been applied) is the "true" residual.... it's what we want to reduce. We then apply the preset as a good "guess" at the solution', are you implying that i should be using DirichletBCs in this example?

Obviously the PresetBC in this example are not 'a good "guess" at the solution'

a